Gopkg.lock

```diff
@@ -1,258 +0,0 @@
-# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Suggestion: extract the Gopkg.lock/Gopkg.toml removal into a separate PR

Reply:
Moved into a separate PR, will merge it into this branch once its in master